README.md

```diff
@@ -48,32 +48,32 @@ flutter-tizen run
 flutter-tizen run --release
 ```
 
-See [Supported commands](doc/commands.md) for all available commands and their basic usage. See `[command] -h` for more information about each command.
-
-#### Notes
-
-- Only the command line interface is available. We have no plan to add IDE extension support.
+- See [Supported commands](doc/commands.md) for all available commands and their basic usage. See `[command] -h` for more information about each command.
+- See [Getting started](doc/get-started.md) to create your first app and try **hot reload**.
 - To **update** the flutter-tizen tool, run `git pull` in this directory.
 
 ## Docs
 
 #### Tizen basics
```

Comment:
Header increased by two levels at a time

Reply:
I find this is not a problem as `####` is just a synonym of HTML's h4 tag.